ITC205_BaseLine_Code/src/PayFineControl.java

```diff
@@ -1,31 +1,30 @@
 public class PayFineControl {
 	
 	private PayFineUI ui;

```

Comment:
D11 - Change:

	private PayFineUI ui;
	private enum CONTROL_STATE { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
	private CONTROL_STATE state;
	
	private library library;
	private member member;;

To:

	private enum ControlState { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };

	private ControlState state;
	private PayFineUi ui;
	private Library library;
	private Member member;

** enums are treated as classes so a line of whitespace follows them generally. Rearranging things slightly and putting the enum first makes for clearer code.